api.js

```diff
@@ -58,12 +58,20 @@ const insertComment = (article_id, newComment, name) => {
 };
 
 const deleteComment = (comment_id) => {
-  console.log(comment_id)
   return axios
     .delete(`https://backend-project-cr4a.onrender.com/api/comments/${comment_id}`)
     .then((data) => {
       return data;
     });
 };
 
-export { getArticles, getArticle, getComments, getTopics, getUsers, insertVotes, insertComment, deleteComment }
+const getTopicalArticles = (topic) => {
+  return axios
+    .get(`https://backend-project-cr4a.onrender.com/api/articles/?topic=${topic}`)
+    .then((data) => {
+      return data;
+    });
```

Comment:
This way is totally fine, but axios has its own way to deal with queries by passing a query object as the second argument of .get method instead of updating the url. e.g. `axios
    .get("https://backend-project-cr4a.onrender.com/api/articles", { params: {topic, sort_by, order ...}})`
This will make it a bit easier, also will be able to used in getArticles instead of writing two separated functions to make it DRY-er